### PR TITLE
Prepare 2.26.0 beta.2

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
Time for another beta.

**What type of PR is this?**
/kind chore

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
NONE
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

File: modules/web/src/app/node-data/component.ts [hunk 1]
```diff
@@ -339,7 +339,7 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
     return provider.includes(this.provider);
   }
 
-  // Source of truth for supported operating system: https://github.com/kubermatic/machine-controller/blob/main/docs/operating-system.md
+  // Source of truth for supported operating system: https://k8c.io/machine-controller/blob/main/docs/operating-system.md
```
Comment (ahmedwaleedmalik) [inline modules/web/src/app/node-data/component.ts:342]: Please revert this change

**Review by ahmedwaleedmalik — CHANGES_REQUESTED** [review]
Just reviewed the code, haven't reviewed anything release related

File: modules/web/src/app/node-data/component.ts (hunk 1 above)
Comment (xrstf) [reply modules/web/src/app/node-data/component.ts:342]: Ah yes, whoops. I saw it 3 times and thought "nice" every time.. 🙈

**Review by Waseem826 — APPROVED** [review]
/approve
